### PR TITLE
Detail view form

### PR DESCRIPTION
**Pull request by yuda1124**

### CODE REVIEW

File: shared/components/Card.js [hunk 1]
```diff
@@ -97,11 +98,13 @@ class Card extends Component {
...
   }
 
-  renderImg(rowData) {
+  renderImg(rowData, sectionID, rowID) {
+
     return (
       <TouchableOpacity onPress={()=>{
-        this.props.getDetailImage(this.props.dataSource.key, 1);
-        Actions.detailView({type: 'replace'});
+        console.log(rowID);
```
Comment (ochanje210) [inline shared/components/Card.js]: please remove console log

File: shared/components/detailHeader.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <View>
+        <View>
+          <Text> {this.props.title} </Text>
+        </View>
+        <View>
+          <TouchableHighlight
+            onPress = {()=>{
+              Actions.pop();
+            }}>
+            <Image source = {require('../resources/btn_close/drawable-mdpi/btn_close.png')}/>
```
Comment (ochanje210) [inline shared/components/detailHeader.js]: please follow ES6 and use import

File: shared/components/detailView.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React, { PropTypes, Component } from 'react';
 import Swiper from 'react-native-swiper';
+import DetailHeader from './detailHeader';
 import { View, Image } from 'react-native';
 
 const styles = {
```
Comment (ochanje210) [inline shared/components/detailView.js]: styles.text is not being used. please delete it if not needed

File: shared/components/detailView.js [hunk 4]
```diff
@@ -22,30 +23,43 @@ const styles = {
...
 
-export default class detailView extends Component {
+export default class DetailView extends Component {
 
   constructor(props) {
     super(props);
   }
 
+  componentDidMount() {
+    this._swiper.scrollBy(this.props.rowID * 1);
```
Comment (ochanje210) [inline shared/components/detailView.js]: great job :D

File: shared/components/detailView.js [hunk 5]
```diff
@@ -22,30 +23,43 @@ const styles = {
...
       </View>
     );
   }
 }
 
-detailView.propTypes = {
-  detailSource: PropTypes.any
+DetailView.propTypes = {
+  detailSource: PropTypes.any,
+  detailIndex: PropTypes.any,
```
Comment (ochanje210) [inline shared/components/detailView.js]: I don't see detailIndex prop being used in this component anywhere. please delete un-used props.

File: shared/containers/detailViewLayout.js [hunk 6]
```diff
@@ -4,7 +4,8 @@ import { connect } from 'react-redux';
 
 const mapStateToProps = (state) => {
   return {
-    detailSource: state.list.detailSource
+    detailSource: state.list.detailSource,
+    detailIndex: state.list.detailIndex
```
Comment (ochanje210) [inline shared/containers/detailViewLayout.js]: please delete detailIndex if not used in the component

File: shared/reducers/list.js [hunk 7]
```diff
@@ -14,9 +14,10 @@ const list = (state = initialState, action) => {
       dataSource: { $set: action.items }
     });
   case types.getDetailImage:
+    console.log(action.index);
```
Comment (ochanje210) [inline shared/reducers/list.js]: please delete console log if you don't need anymore

File: shared/components/detailView.js (hunk 5 above)
Comment (yuda1124) [inline shared/components/detailView.js]: Okay i will

File: shared/reducers/list.js (hunk 7 above)
Comment (yuda1124) [inline shared/reducers/list.js]: Okay kk

**Comment by ochanje210** [comment]
moved to https://github.com/sv-bootcamp/goober-mobile/pull/67

**Pull request closed without merging**